Makefile

```diff
@@ -32,3 +32,6 @@ docs-test: python-build
 fmt:
 	cargo fmt
 	$(MAKE) -C languages/python fmt
+
+clippy:
+	cargo clippy --all-features --all-targets -- -D warnings
```

Comment:
if you really want this to catch _everything_, I'd recommend prefixing it with `touch polar/src/lib.rs && cargo clippy...`